lib/src/result.dart

```diff
@@ -37,7 +37,7 @@ abstract class Result<T> {
   /// The result of the transformation is a sink that only forwards [Result]
   /// values and no error events.
   static const StreamSinkTransformer<Object, Result> captureSinkTransformer =
-      const StreamSinkTransformer.fromStreamTransformer(
```

Comment:
@jmesserly Why isn't this inferred? Same with the other changes in this file and `stream_sink_transformer.dart`.